onager/backends/_backend.py

```diff
@@ -16,13 +16,18 @@ def __init__(self):
 
         self.task_id_var = r'$TASK_ID'
 
+    def _get_body(self, tasks_file, args):
```

Comment:
Allows us to do more complicated things than format the string, but leaves the default